File: trunk/Templates/Examples/NHibernate/CSharp/NHibCsSample.Data/ManagerObjects/LineItemManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using NHibCsSample.Generated.BusinessObjects;
using NHibCsSample.Generated.Base;

namespace NHibCsSample.Generated.ManagerObjects
{
    public partial interface ILineItemManager : IManagerBase<LineItem, string>
    {
	}
	
	partial class LineItemManager : ManagerBase<LineItem, string>, ILineItemManager
    {
	}
}

File: Projects/Framework-Samples/NHibernate/VisualBasic/Sample.Data/ManagerObjects/LineItemManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports Sample.Data.Generated.BusinessObjects
Imports Sample.Data.Generated.Base

Namespace Sample.Data.Generated.ManagerObjects
	'Public Partial Interface ILineItemManager
	'	Inherits IManagerBase(Of LineItem, System.String)
	'End Interface

	Partial Class LineItemManager
		Inherits ManagerBase(Of LineItem, System.String)
		Implements ILineItemManager
	End Class
End Namespace
